### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by Fombi-Favour**
In this activity, we:
- created a database
- created a table
- inserted data into the table
- filter the table with queries

## Queries
- >Find all animals whose name ends in "mon".
project1-sc1
- >List the name of all animals born between 2016 and 2019.
project1-sc2
- >List the name of all animals that are neutered and have less than 3 escape attempts.
project1-sc3
- >List the date of birth of all animals named either "Agumon" or "Pikachu".
project1-sc4
- >List name and escape attempts of animals that weigh more than 10.5kg
project1-sc5
- >Find all animals that are neutered.
project1-sc6
- >Find all animals not named Gabumon.
project1-sc7
- >Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
project1-sc8

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '01-01-2016' AND '12-31-2019';
+SELECT name FROM animals WHERE neutered=true AND escape_attempts<=3;
```
Comment (shahadat3669) [inline queries.sql]: 👍 Great job! This SQL query effectively retrieves the names of animals from the "animals" table that are both neutered and have either less than or equal to 3 escape attempts. By using the `WHERE` clause with the conditions `neutered=true` and `escape_attempts<=3`, the query ensures that it includes animals that are neutered and those with up to 3 escape attempts, providing a concise summary of the desired information, which encompasses animals with 0, 1, 2, or 3 escape attempts. 

- [ ] However, please consider the project requirement, which states: "List the name of all animals that are neutered and have less than 3 escape attempts." To align with this requirement, you should adjust your query to include animals with fewer than 3 escape attempts. Additionally, if needed, update the Output Screenshot in the Pull Request Description to reflect the correct results. Keep up the excellent work! 👏🐾🐕

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @Fombi-Favour, 

Great job so far! 👏 You've done well, and you're almost there for the next project ⏭️, but there are a few issues 🐛 that you still need to work on.




## Don't Worry 🙏

It's okay to have bugs in your code. 🆗 Please see this code review as an opportunity to learn practical coding skills and discover new tips that can improve your code quality 💯 and help you **get even better** 💼.

**I will guide you on how to address these areas down below 👇.**

## Highlights
✅ put correct sql commands in the correct files
✅ create table inside called `animals` with required columns
✅ add the required datas in the data.sql file


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

## ⚠️ WARNING ⚠️

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Fombi-Favour) [reply queries.sql]: Thanks for the feedback 🙏

File: README.md [hunk 2]
```diff
@@ -1 +1,161 @@
...
+
+I would like to thank  everybody most especially the Microverse staffs for making it possible to work on the "vet-clinic" project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed._
```
Comment (zeff96) [inline README.md]: - [ OPTIONAL ] Hello there! 👋 I've taken a look at your project, and I've noticed a slight inconsistency with your licensing. 📜 In the root of your project, you have a file named LICENSE.md. However, in your README file, you're referencing a file called LICENSE which, as it stands, doesn't exist in the root directory of your project. 📂

- To align these references properly, please adjust the relative path in your README to match the file named LICENSE.md located in the root of your project. This ensures that users can easily locate and understand your project's licensing terms without any confusion. 📝🔍
- Maintaining a clear and accurate project structure and documentation is essential for a smooth user experience and adherence to licensing requirements. 🚀👨‍💻

Screenshot from 2023-10-23 13-42-39

**Review by zeff96 — APPROVED** [review]
### Status: APPROVED :green_circle: 
Hi @Fombi-Favour , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congrats

## Highlights
:heavy_check_mark: Correct GitFlow
:heavy_check_mark: Descriptive pull request
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by V-Blaze — COMMENTED** [review]
Hi @Fombi-Favour 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved-accepted


## Highlights
- Table created correctly and correct data inserted💯
-  Querry result screenshots added to pull requests👍
- Queries written correctly 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**